.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "definitions"]
+	path = definitions
+	url = git@github.com:hmcts/sscs-ccd-definitions.git
```

Comment:
does this mean we always will checkout master definition ? If we had to deploy local ccd definition changes how do we go about it ?

Reply:
It would clone the entire defs into the folder and then you can change them as normal I believe